src/testers.jl

```diff
@@ -129,9 +132,9 @@ function test_frule(
         # Correctness testing via finite differencing.
         dΩ_fd = _make_jvp_call(
             fdm,
-            (f, xs...) -> f(deepcopy(xs)...; deepcopy(fkwargs)...),
+            (f, xs...) -> call_on_copy(f, xs...),
```

Comment:
Isn't this just the same as:
```suggestion
            call_on_copy,
```